compiler/rustc_hir_typeck/src/method/suggest.rs

```diff
@@ -369,6 +369,7 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
         };
         if let Some(file) = file {
             err.note(format!("the full type name has been written to '{}'", file.display()));
+            err.note(format!("consider using --verbose to print full type name to console"));
```

Comment:
sorry, i really feel like this back and forth is annoying, but this isn't grammatical 
 -- one more round please? 😭

```suggestion
            err.note(format!("consider using `--verbose` to print the full type name to the console"));
```

Reply:
Oh it is fine! :-)

Reply:
Yay! Done. Do not worry if you think it is not good yet :)